app/src/main/java/com/infomaniak/mail/data/cache/mailboxContent/ThreadController.kt

```diff
@@ -209,13 +205,13 @@ class ThreadController @Inject constructor(
         //region Edit data
         fun upsertThread(thread: Thread, realm: MutableRealm): Thread = realm.copyToRealm(thread, UpdatePolicy.ALL)
 
-        suspend fun fetchIncompleteMessages(
+        suspend fun fetchMessagesHeavyData(
             messages: List<Message>,
             mailbox: Mailbox,
             okHttpClient: OkHttpClient? = null,
             realm: Realm,
```

Comment:
Inverse these lines to put optional parameter at the end
Do the same for updateFailedFolders